src/lib.rs

```diff
@@ -203,6 +212,31 @@ impl TcpStream {
             Ok(ret)
         }
     }
+
+    pub fn local_addr(&self) -> Result<SocketAddr> {
```

Comment:
` [0u8; 16] ` is better than `vec`

[ pub fn get_local(&self) -> io::Result<SocketAddr>](https://github.com/second-state/wasmedge-quickjs/blob/c5ffed13931fd5d4e99fd6e02aa1e570f40133dc/src/event_loop/wasi_sock.rs#L247)